### PR TITLE
Adding Twitter

### PR DESCRIPTION
**Pull request by JessicaGarson**
Add Twitter's Account Activity API to the directory

### CODE REVIEW

File: src/pages/docs/webhook-directory.md [hunk 1]
```diff
@@ -56,8 +56,9 @@ New webhooks are created and improved every day. Please [contribute](/docs/how-t
   {% webhook-entry provider="Tiktok" hash="sha256" encode="hex" rotation="❌" versioning="❌" timestamp="✅" link="https://developers.tiktok.com/doc/webhooks-verification" /%}
   {% webhook-entry provider="TrendMicro Conformity" hash="sha256" encode="hex" rotation="❌" versioning="❌" timestamp="❌" link="https://cloudone.trendmicro.com/docs/conformity/webhook-communication/" /%}
   {% webhook-entry provider="Twilio" hash="sha1" encode="base64" rotation="❌" versioning="❌" timestamp="❌" link="https://www.twilio.com/docs/usage/security#validating-requests" /%}
+  {% webhook-entry provider="Twitter Account Activity API" hash="SHA265" encode="base64" rotation="❌" versioning="❌" timestamp="✅" link="https://developer.twitter.com/en/docs/twitter-api/premium/account-activity-api/overview" /%}
```
Comment (sudobinbash) [inline src/pages/docs/webhook-directory.md]: Hi @JessicaGarson. Thanks for contributing <3 
Can we change the hash to `sha256`? 
(the rest looks good)

**Review by sudobinbash — CHANGES_REQUESTED** [review]
Hi @JessicaGarson. Thanks for contributing <3
Can we change the hash to sha256?
(the rest looks good)

File: src/pages/docs/webhook-directory.md (hunk 1 above)
Comment (JessicaGarson) [reply src/pages/docs/webhook-directory.md]: Yes!

Comment (JessicaGarson) [reply src/pages/docs/webhook-directory.md]: Just made that change

**Comment by JessicaGarson** [comment]
Made these changes @sudobinbash, let me know if there is anything more you need

File: src/pages/docs/webhook-directory.md (hunk 1 above)
Comment (sudobinbash) [reply src/pages/docs/webhook-directory.md]: Hi @JessicaGarson . It seems like you replaced the wrong field (encode) with sha256:

```
{% webhook-entry provider="Twitter Account Activity API" hash="SHA265" encode="sha256" rotation="❌" versioning="❌" timestamp="✅" link="https://developer.twitter.com/en/docs/twitter-api/premium/account-activity-api/overview" /%}
```

I should look like this:

```
{% webhook-entry provider="Twitter Account Activity API" hash="sha256" encode="base64" rotation="❌" versioning="❌" timestamp="✅" link="https://developer.twitter.com/en/docs/twitter-api/premium/account-activity-api/overview" /%}
```

**Review by sudobinbash — APPROVED** [review]

**Comment by sudobinbash** [comment]
Hi @JessicaGarson   👋

Thanks for contributing to webhooks.fyi.
We're launching our site on [Product Hunt tomorrow](https://www.producthunt.com/posts/webhooks-fyi) and I'd love to include our contributors as makers (to spread the love and street creds).

Please sign up to [producthunt.com](https://producthunt.com) and send me your profile url here if you would like to be included and I'll take care of the rest.

ty,
Fred

**Comment by JessicaGarson** [comment]
Hi,

The past few days were a bit crazy on my end, but my product hunt profile
is here: ***@***.***

Best,

Jessica

**Pull request merged**